frontend/src/grpcweb/middlewares/authInterceptorMiddleware.ts

```diff
@@ -4,6 +4,13 @@ import { ClientMiddleware } from "nice-grpc-web";
 import { useAuthStore } from "@/store";
 import { router } from "@/router";
 
+export type IgnoreErrorsOptions = {
+  /**
+   * if set to true, will NOT handle specified status codes.
```

Comment:
nit: the comment is a bit off.  Not a boolean

Reply:
updated.